src/components/Avatar/Avatar.svelte

```diff
@@ -0,0 +1,85 @@
+<script lang="ts">
+    import type { Action } from "svelte/action";
+    
+    // Note: consider `./types.ts`
+    interface AvatarProps {
+        // initials
+        initials?: string;
+        initialsFill?: string;
+        initialsFontSize?: number;
+        initialsBase?: string;
+        initialsRest?: string;
+
+        // image
+        imageBase?:string;
+        imageSrc?: string;
+        imageAlt?: string;
+        imageFallback?: string;
+        imageAction?: Action<HTMLElement, string>;
+        imageActionParams?: string;
+        imageRest?: string;
+
+        // figure
+        figureBase?: string;
+        figureBackground?: string;
+        figureWidth?: string;
+        figureBorder?: string;
+        figureRounded?: string;
+        figureShadow?: string;
+        figureCursor?: string;
+        figureRest?: string;
+    }
+
+    let {
+        // initials
+        initials = "AB",
+        initialsFill = "fill-token",
+        initialsFontSize = 150,
+        initialsBase = "w-full h-full",
+        initialsRest = "",
+
+        //image
+        imageBase = "w-full object-cover",
+        imageSrc = "",
+        imageAlt = "",
+        imageFallback = "",
+        imageAction = () => {},
+        imageActionParams = "",
+        imageRest = "",
+
+        // figure
+        figureBase = "flex aspect-square text-surface-50 font-semibold justify-center items-center overflow-hidden isolate",
```

Comment:
We might want split a couple more of the base values to style props, like `text-` and `font-`. The others should be fine though!